cime_config/machines/cmake_macros/gnu_crusher.cmake

```diff
@@ -0,0 +1,19 @@
+if (compile_threaded)
+  string(APPEND CFLAGS " -fopenmp")
+  string(APPEND FFLAGS " -fopenmp")
+  string(APPEND CXXFLAGS " -fopenmp")
+  string(APPEND LDFLAGS " -fopenmp")
+endif()
+
+string(APPEND FFLAGS " -fallow-argument-mismatch -Wno-implicit-interface -fallow-invalid-boz -fno-tree-pta ")
```

Comment:
`Depends.gnu.cmake` has `-fno-tree-pta` for one ICE file. Need to apply to all files?

Reply:
I did not add this option but will check if this can be eliminated or at least reduced to a narrower scope.

Reply:
`-fallow-argument-mismatch` is now on master in cmake_macros/gnu.cmake . It can be omitted here.

Same for `-fallow-invalid-boz -fno-tree-pta`: they are in Depends.gnu.cmake .

So this can be updated to
```
string(APPEND FFLAGS " -Wno-implicit-interface")
```